### PR TITLE
fmt: clang-format the code in psi4/src

### PR DESCRIPTION
**Pull request by andysim**
## Description
clang-format the code in psi4/src

## Questions
- [ ] If merging this PR will hinder your research b/c you have local edits, protest here or in #1144

N.B. Pay particular attention to `read_options.cc`, which I formatted manually to preserve docstring formatting.

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: psi4/src/export_mints.cc [hunk 1]
```diff
@@ -1232,39 +1226,47 @@ void export_mints(py::module& m) {
...
-        .def("ftrue_atomic_number", &Molecule::ftrue_atomic_number, "Gets atomic number of "
-             "*atom* from element (0-indexed including dummies)", py::arg("atom"))
+        .def("true_atomic_number", &Molecule::true_atomic_number,
+             "Gets atomic number of "
+             "*atom* from element (0-indexed without dummies)",
+             py::arg("atom"))
+        .def("ftrue_atomic_number", &Molecule::ftrue_atomic_number,
+             "Gets atomic number of "
+             "*atom* from element (0-indexed including dummies)",
+             py::arg("atom"))
```
Comment (loriab) [inline psi4/src/export_mints.cc]: c-f unnecessarily humoring our crazy merge-string docstring

File: psi4/src/export_mints.cc [hunk 2]
```diff
@@ -1232,39 +1226,47 @@ void export_mints(py::module& m) {
...
         .def("label", &Molecule::label,
              "Gets the original label of the *atom* as given in the input file (C2, H4)"
-             "(0-indexed without dummies)", py::arg("atom"))
+             "(0-indexed without dummies)",
+             py::arg("atom"))
         .def("flabel", &Molecule::flabel,
              "Gets the original label of the *atom* arg0 as given in the input file (C2, H4)"
-             "(0-indexed including dummies)", py::arg("atom"))
+             "(0-indexed including dummies)",
+             py::arg("atom"))
```
Comment (loriab) [inline psi4/src/export_mints.cc]: ditto these two

**Comment by loriab** [comment]
Ambitious to hand-format read_options :-)

What's wrong with the c-f of it, though? I can see that things like the below might need adjustment, but at a glance, it looked pretty respectful. line-break formatting isn't conveyed to the manual anyways.

```
        options.add_str("CFOUR_BASIS", "SPECIAL",
                        "STO-3G 3-21G 4-31G 6-31G 6-31G* 6-31G** 6-311G 6-311G* 6-311G** DZ DZP TZ TZP TZ2P PVDZ PVTZ "
                        "PVQZ PV5Z PV6Z PCVDZ PCVTZ PCVQZ PCV5Z PCV6Z AUG-PVDZ AUG-PVTZ AUG-PVTZ AUG-PVQZ AUG-PV5Z "
                        "AUG-PV6Z D-AUG-PVDZ D-AUG-PVTZ D-AUG-PVQZ D-AUG-PV5Z D-AUG-PV6Z cc-pVDZ cc-pVTZ cc-pVQZ "
                        "cc-pV5Z cc-pV6Z cc-pCVDZ cc-pCVTZ cc-pCVQZ cc-pCV5Z cc-pCV6Z PWCVDZ PWCVTZ PWCVQZ PWCV5Z "
                        "PWCV6Z PwCVDZ PwCVTZ PwCVQZ PwCV5Z PwCV6Z svp dzp tzp tzp2p qz2p pz3d2f 13s9p4d3f WMR ANO0 "
                        "ANO1 ANO2 EVEN_TEMPERED SPECIAL");
```

```
    if (name == "FINDIF" || options.read_globals()) {
        /*- MODULEDESCRIPTION Performs finite difference computations of energy derivative, with respect to nuclear
           displacements
            for geometry optimizations and vibrational frequency analyses, where the required analytical derivatives are
           not
            available. -*/
```

**Comment by loriab** [comment]
I'm glad to fix these comments, btw. Mostly just marking them.

**Comment by andysim** [comment]
The problem with c-f in read_options was comments like
```
/*- This is a multiline
      string -*/
```
becoming
```
/*- This is a
 * multiline string -*/
```

**Pull request merged**